tests/qam-minimal/update_minimal.pm

```diff
@@ -31,7 +31,7 @@ sub run {
 
     capture_state('between-after');
 
-    assert_script_run("zypper lr | grep test-minimal");
+    assert_script_run("zypper lr | grep TEST_");
```

Comment:
can you elaborate why you select "TEST_" as a pattern ?

Reply:
Because that is the variable used in `add_test_repositories` from lib/qam.pm

Reply:
ah I got confused , so replacing "test-minimal" in install-update.pm with "MAINT_TEST_REPO"  and checking for existence of "TEST_" instead of "test-minimal" are not related to each other ?

Reply:
Those changes are connected. 

1. Function `add_test_repositores` gets the repository addresses from `MAIN_TEST_REPO` variable and adds multiple "TEST_[0-9]+" repositories in the system.
2. Function in install_update.pm wants to use this function, but gets the repo path in another variable, hence the `set_var`
3. The new repo is not longer called "test-minimal", hence the pattern has to be changed in the update_minimal.pm

This is needed for poo#36583.